praetorian_cli/ui/aegis/commands/proxy.py

```
@@ -0,0 +1,337 @@
+    elif subcmd == 'stop':
+        _handle_stop(menu, args[1:])
+    else:
+        _handle_start(menu, args)
+
+
+def _handle_start(menu, args):
+    """Parse port + optional -u, validate agent, launch proxy."""
+    colors = getattr(menu, 'colors', DEFAULT_COLORS)
+
+    if not menu.selected_agent:
+        menu.console.print(f"\n  No agent selected. Use 'set <id>' to select one.\n")
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)